integrationtest.py

```diff
@@ -45,13 +48,29 @@ def deploy_and_run_smoketest_pipeline(
     # our pipelines repository
     calcdir = lambda path: os.path.join(cc_pipelines_repo_dir, path)
 
-    pipeline_definition_file = calcdir('definitions/test/cc-smoketest.yaml')
     template_path = calcdir('templates')
     template_include_dir = cc_pipelines_repo_dir
     pipeline_name = 'cc-smoketest'
     job_name = 'cc-smoketest-master-head-update-job'
 
-    pipeline_definition = parse_yaml_file(pipeline_definition_file, as_snd=False)
+    pipeline_definition = parse_yaml_file(pipeline_definition_file, )
```

Comment:
according to PEP-8, it is a discouraged practice to write a trailing comma in single-lined argument lists. Also: why did you change this back to a SND? This does not belong to the semantics of this commit either way.

Reply:
thinking of it: this cannot work, can it? pipeline_deifnition_file is no longer defined, as line 48 is removed. so this looks like a bug to me. did you actually test this?